app/components/market/submit/market-application.model.ts

```diff
@@ -0,0 +1,83 @@
+import { NcjParameter } from "app/models";
+import { ObjectUtils, log } from "app/utils";
+import * as inflection from "inflection";
+
+export enum Modes {
+    None,
+    NewPoolAndJob,
+    OldPoolAndJob,
```

Comment:
Nitpick: "ExistingPoolAndJob" instead of "OldPoolAndJob". Thoughts?

Reply:
done